protein_ligand/README.md

```diff
@@ -135,6 +136,27 @@ We also accept non-ranked submissions, which we will not formally judge. These a
 
 If multiple submissions are incorrectly provided as "ranked" by a single participant, we will judge only one of them; likely this will be the first submitted, but it may be a random submission.
 
+**Submission format**: Your submissions must consist of a completed submission description following [our format](stage2_submission_template.txt) (with a file name beginning with the characters `PHIP2_2` and ending with `.txt`) and a .tar.gz `pose.tgz` file providing your pose predictions compressed and archived in a single directory. This file is used to submit one set of predicted protein-ligand poses, where up to five poses are permitted for each protein-ligand pair (if multiple poses are submitted, subsequent poses are given decreasing weight in judging). Each pose tgz file must contain, for each ligand, a minimum of one and up to 5 protein structure PDB files and 5 corresponding ligand poses (in MDL `.mol`, format, `.sdf` format, or `.mol2` format) predicted by a method described in the separately uploaded pose prediction description file.  
```

Comment:
I think how the evaluation will be handled for 5 poses per ligand is ambiguous. What does it mean to give "decreasing weight of judging"? Will the pose RMSD calculated as weighted average based on the rank of the pose?
If we decide to allow multiple poses to be submitted, I think the best way for method comparison would be to formally evaluate only the best pose (the pose with the lowest RSMD). Otherwise, I worry the errors from the alternative poses will create a noise that masks the importance of the best pose.

Reply:
The current file names for poses does not request any information about the order of predictions. I think it would be necessary to add an index to indicate the priority of each prediction if "subsequent poses are given decreasing weight". 
However, I think it would be better to treat all poses equally and just judge the best pose or only allow for 1 pose submission.

Reply:
My thought would be that we would do two analyses:
1) Analyze only the best scoring (first) pose
2) Look at the top five poses and analyze the lowest RMSD pose

Reply:
I think it makes sense to look at the top five. If we already have the scripts to analyze these we might as well give ourselves the option of looking at this.